src/components/AppInstanceLabel.js

```diff
@@ -25,7 +35,18 @@ const AppInstanceLabel = React.memo(
             <AppIcon app={app} />
           </div>
         )}
-        <AppName>{app ? app.name : 'Unknown'}</AppName>
+        <span
+          css={`
+            display: block;
+            ${above('medium') &&
+              `
+                display: inline;
+                margin-right: 10px;
```

Comment:
```suggestion
                margin-right: ${1.25 * GU}px;
```

But maybe 1gu would work here?

Reply:
I believe it does (it's the space between the app name and address / short name) :
<img width="402" alt="Screen Shot 2020-04-08 at 12 04 42 AM" src="https://user-images.githubusercontent.com/26014927/78743425-9a272500-792c-11ea-8684-21adaf7fc2b7.png">